zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/CassandraSpanStore.java

```diff
@@ -305,8 +305,8 @@
     UUID endUUID;
   }
 
-  TimestampRange timestampRange(QueryRequest request) {
-    long oldestData = Math.max(System.currentTimeMillis() - indexTtl * 1000, 0); // >= 1970
+  TimestampRange timestampRange(QueryRequest request, int indexTtl) {
```

Comment:
Do we truly need to pass indexTtl as param? I see this is a class attribute: https://github.com/openzipkin/zipkin/pull/3367/files#diff-d85e858924df5caed00b20b67deac8389b768227fc7fdf8f7575884ae941a663L54

Reply:
@jcchavezs the indexTtl value comes from a static method and then gets set to a final class attribute. To create a test I need to either:
1) Set the property using reflection for the test.
2) Mock a static method.
3) Pass the indexTtl as a param.

I chose 3 since it seemed as the least amount of hassle. Tell me which option you would prefer and I'll implement it.
Cheers!